starknet_devnet/blueprints/gateway.py

```diff
@@ -31,6 +31,9 @@ async def add_transaction():
         )
         response_dict["class_hash"] = hex(contract_class_hash)
 
+    if tx_type == TransactionType.DEPLOY_ACCOUNT:
```

Comment:
Hey, I've tried to run `starknet.py` with devnet from this branch and it is returning an `Invalid tx_type: DECLARE` error. I believe it is caused by this `if` statement. After changing `if` -> `elif` it should work. Many thanks for updating to Cairo 0.10.1 🤝

Reply:
Yes, thank you for dropping an eye, this last commit is marked as WIP and has several TODOs, though. The last commit with a green check (meaning runnable and all tests passing) is `86fd2bf552a5814f1ef2a87aebfbb4c292eddd29`.

Reply:
Oh, right 😅 I didn't notice the check. Thanks!